### DISCUSSION TITLE
Syntheny plot

### DISCUSSION THREAD

**Discussion by vladsavelyev** [ablab/quast#171] [Ideas]
Read nucmer output and build a syntheny plot:

![pasted image at 2017_02_14 03_17 pm](https://cloud.githubusercontent.com/assets/1575412/22948188/9bd85600-f30e-11e6-8968-2b8c01f5856e.png)

Consider known tools first, there are plenty of those.

Create both PDF and JS versions.

Thanks @snurk

**Comment by mikolmogorov** [comment]
In fact, I would love to see this feature in QUAST too :) You can use MashMap (it's pretty fast)  - https://github.com/marbl/MashMap

**Comment by sjackman** [comment]
I generate a dotplot like this using `minimap2` and `minidot` from `miniasm`.
See https://github.com/lh3/miniasm/blob/1552e6f9a22ae3a5285696fd8dc9882192288083/Makefile#L19
